config/ecflow_template.cfg

```diff
@@ -0,0 +1,86 @@
+[MAIN]
+MakeDummyScripts: False 
+NSuites: 1
+DefFile: 
+RepeatDay: 1
+
+[S0]
+Name: processes
+NFamilies: 1
+NTasks: 0
+NVariables: 11
+VariableName0: ECF_JOB_CMD
+VariableValue0: qsub %%ECF_JOB%%
+VariableName1: ECF_KILL_CMD
+VariableValue1: qdel %%ECF_RID%%
+VariableName2: ECF_INCLUDE
+VariableValue2: 
+VariableName3: ECF_HOME
+VariableValue3: 
+VariableName4: USER_EMAIL
+VariableValue4: 
+VariableName5: CORES
+VariableValue5: 
+VariableName6: CFG_FILE
+VariableValue6: 
+VariableName7: MET_LOC
```

Comment:
A variable name added for the working_python configuration file. Appropriate changes made to the reorder, regrid, combine, run_vic, percent_plot and vic2netcdf .ecf files.